Wikipedia/Code/WMFWelcomePageViewController.swift

```diff
@@ -60,38 +77,75 @@ class WMFWelcomePageViewController: UIPageViewController, UIPageViewControllerDa
         
         setViewControllers([pageControllers.first!], direction: direction, animated: true, completion: nil)
 
-        addGradient()
+        view.backgroundColor = UIColor(0xffffff)
+        
+        configureAndAddNextButton()
+        configureAndAddSkipButton()
         
         if let scrollView = view.wmf_firstSubviewOfType(UIScrollView.self) {
             scrollView.clipsToBounds = false
         }
     }
+    
+    fileprivate func configureAndAddNextButton(){
+        nextButton.translatesAutoresizingMaskIntoConstraints = false
+        nextButton.addTarget(self, action: #selector(nextButtonTapped), for: .touchUpInside)
+        nextButton.isUserInteractionEnabled = true
+        nextButton.titleLabel?.adjustsFontSizeToFitWidth = true
+        nextButton.setTitle(CommonStrings.nextTitle, for: .normal)
+        nextButton.setTitleColor(.wmf_blue, for: .normal)
+        // TODO: figure out what to do about 0xA2A9B1 - not sure what to name this gray based on the other gray names - we already have  'light(er|est)Gray'
```

Comment:
Ask Carolyn if we can use one of the existing gray colors, otherwise `lightererGray`

Reply:
lol :)

Reply:
Fixed. She said it could just use lighterGray.